ocean-client/src/programs/vault-maxi-program.ts

```diff
@@ -306,6 +321,23 @@ export class VaultMaxiProgram extends CommonProgram {
         if(!this.settings.reinvestThreshold || this.settings.reinvestThreshold <= 0) {
             return false
         }
+
+        const utxoBalance= await this.getUTXOBalance()
+        if(utxoBalance.gt(this.settings.reinvestThreshold) && utxoBalance.gt(1)) {
+            const amount= utxoBalance.minus(1)
+            console.log("converting " + amount + " UTXOs to token ")
+            const tx= await this.utxoToOwnAccount(amount)
+            //@Krysh ok if we flag this also as reinvest? or different transaction? if we have a timeout after this, we don't need a cleanup
```

Comment:
fine for me to flag that as reinvestment, because technically it is. Flagging it as anything else would just increase complexity for no benefit at all